app/assets/javascripts/course/components/CoursePage.js

```diff
@@ -114,10 +118,62 @@ class CoursePage extends React.Component {
     return this.state.courseSidebar.sections.find(byId)
   }
 
+  getComponentIndex(subsection, component) {
+    return subsection.components.indexOf(component)
+  }
+
+  isLastComponent(subsection, component) {
+    var components = subsection.components
```

Comment:
dont use var, use `let` if you want the variable to be mutable, `const` if you want it not to be.